opsramp/base.py

```diff
@@ -147,7 +206,15 @@ def process_result(url, resp):
             )
             raise RuntimeError(msg)
         try:
-            return resp.json()
+            data = resp.json()
+            # Some GET requests return paginated output. If all the data fits
+            # in one page, return just the contents of the "results" list,
+            # otherwise, we need to do an assembly job to collate the entire
+            # list of results from all pages and return the full list.
+            if resp.request.method == "GET" and "nextPage" in data.keys():
+                return ApiObject.collate_pages(resp.request, data=data)
+            else:
+                return data
         except: # noqa
```

Comment:
I don't really like blanket except (neither does flake8 hence the #noqa i guess) and especially now that there is a lot more code in the "try" block that could raise exceptions.

Reply:
Yep, makes sense.

Reply:
This error handler has been fixed in head of master by #57 so i'm closing this comment on that basis, assuming this PR will get rebased before merge anyway.